packages/react-native-sdk/src/components/call/CallTopView/CallTopView.tsx

```diff
@@ -40,34 +42,42 @@ export const CallTopView = ({
   title,
   style,
 }: CallTopViewProps) => {
+  const callingState = useCallCallingState();
   const [headerHeight, setHeaderHeight] = useState<number>(0);
+  const isCallReconnecting = callingState === CallingState.RECONNECTING;
 
   const onLayout: React.ComponentProps<typeof View>['onLayout'] = (event) => {
     const { height } = event.nativeEvent.layout;
     setHeaderHeight(height);
   };
 
   return (
-    <>
+    <View style={[styles.container, style]}>
       {/* Component for the background of the CallTopView. Since it has a Linear Gradient, an SVG is used to render it. */}
-      <TopViewBackground
-        height={headerHeight}
-        width={'100%'}
-        style={[styles.background, style]}
-      />
-      <View style={[styles.container, style]} onLayout={onLayout}>
+      <TopViewBackground height={headerHeight} width={'100%'} />
+      <View style={styles.topView} onLayout={onLayout}>
         <View style={styles.leftElement}>
           {onBackPressed && (
             <Pressable
-              style={[theme.icon.md, styles.backIcon]}
+              style={({ pressed }) => [
+                theme.icon.md,
+                styles.backIcon,
+                { opacity: pressed ? 0.2 : 1 },
+              ]}
               onPress={onBackPressed}
             >
               <Back color={theme.light.static_white} />
             </Pressable>
           )}
         </View>
         <View style={styles.centerElement}>
-          {title && <Text style={styles.title}>{title}</Text>}
+          {title ? (
+            <Text style={styles.title}>{title}</Text>
+          ) : (
+            isCallReconnecting && (
+              <Text style={styles.title}>Reconnecting...</Text>
```

Comment:
```
              <Text style={styles.title}>Reconnecting...</Text>
```

lets add this text to translations please

Reply:
Done